trackma/ui/cli.py

```diff
@@ -362,6 +365,39 @@ def do_list(self, args):
         # Show the list in memory
         self._make_list(self.sortedlist)
 
+    def do_list_json(self, args):
+        """
+        Print list of shows in json format.
```

Comment:
Should mention that each entry is printed as a JSON line on its own (instead of an array). At least right now. 

If we were to consider things like the scoring format, it may be useful to also report that in a meta object on its own, in which case the question follows whether the entries should instead be wrapped in an array to build an object as follows:

```json
{ "account": "string", "api": "string", "has_progress": false, "scoring_format": "enum", "entries": [{…}] }
```

I don't now how the scoring format options in anilist are implemented currently in various parts of the code or whether it makes more sense to instead specify the `max_score` to be of either 3, 5, or 10 (and use the latter for 10, 20, and 100-point schemas).

And that it is intended to be used by third-party tools.

Reply:
For my particular usage, each entry being on its own line is better. In case another shell program wants to use this, I think it might be better to leave it like this.

The scoring format is something I was simply wondering about, but I think is a good idea to investigate.

Reply:
I'll leave this unresolved for @z411 to read.

Reply:
Kind reminder.
@z411